### PR TITLE
Updated KMODS for release of CyaraOS 8.9 & 9.3

### PR DESCRIPTION
**Pull request by PatrickCoakley23**
All kmods built successfully and tested on CyaraOS

I also added the cyaraos-release files, just so we have history of changes.

Signed: Patrick Coakley

### CODE REVIEW

**Review by jdieter — CHANGES_REQUESTED** [review]

File: cyaraos-release/cyara-release-el9.spec [hunk 1]
```diff
@@ -0,0 +1,365 @@
+%global distro  CyaraOS
```
[the remark this answers is not in the document]
Comment (PatrickCoakley23) [reply cyaraos-release/cyara-release-el9.spec]: Just changed that. Its `cyaraos-release-el9` on osbuilder, just made an error when saving it to this repo

**Review by jdieter — CHANGES_REQUESTED** [review]
Can you please also remove any other rpms in the repo?  And maybe make sure that RPM wildcards are in `.gitignore`

**Comment by PatrickCoakley23** [comment]
> Can you please also remove any other rpms in the repo? And maybe make sure that RPM wildcards are in `.gitignore`

That is done now

**Review by jdieter — APPROVED** [review]
Ok, this looks good.  Please squash and merge.

**Pull request merged**